packages/fiber/package.json

```diff
@@ -30,12 +30,10 @@
   "main": "dist/react-three-fiber.cjs.js",
   "module": "dist/react-three-fiber.esm.js",
   "types": "dist/react-three-fiber.cjs.d.ts",
-  "react-native": "native/dist/react-three-fiber-native.cjs.js",
   "sideEffects": false,
   "preconstruct": {
     "entrypoints": [
-      "index.tsx",
-      "native.tsx"
+      "index.tsx"
     ]
   },
```

Comment:
Preconstruct isn't picking up the native stuff. Not sure how we can include them in a single build step.

![image](https://user-images.githubusercontent.com/23324155/144732252-097694cf-6ff6-4d2d-85a6-23a3d94b0ddf.png)